jpa/pom.xml

```diff
@@ -39,25 +39,12 @@
 
     <modules>
         <module>hibernate6</module>
-    </modules>
-
-    <profiles>
-        <profile>
-            <id>ee8.build.profile</id>
-            <activation>
-                <property>
-                    <name>!skip.ee8</name>
-                </property>
-            </activation>
-            <modules>
-                <!-- TODO once moved to jakarta.* move to main module set-->
-                <module>subsystem</module>
-                <!-- TODO once moved to jakarta.* move to main module set-->
-                <module>spi</module>
-                <!-- TODO once moved to jakarta.* move to main module set-->
-                <module>eclipselink</module>
-            </modules>
-        </profile>
-    </profiles>
+        <module>subsystem</module>
+        <module>spi</module>
+<!-- TODO enable eclipselink module after resolving 
```

Comment:
@scottmarlow Is there a JIRA to track doing this?

Reply:
Added commit for EclipseLink, the rest of the changes are squashed into other commit.

Reply:
I look forward to creating the separate project to host feature packs for adding EclipseLink and possibly some other persistence providers.

Reply:
Sorry, the jira is https://issues.redhat.com/browse/WFLY-17123